.azure/azure-pipelines.ci.yml

```diff
@@ -203,38 +203,92 @@ stages:
 # Performance Tests
 #
 
-- stage: performance
-  displayName: Performance Testing
+- stage: remoteperformance
+  displayName: Remote Performance Testing
   dependsOn:
   - build_windows
   - build_linux
   jobs:
-  - template: ./templates/run-performance.yml
+  - template: ./templates/run-remote-performance.yml
     parameters:
       pool: MsQuic-Win-Perf
       platform: windows
-      tls: stub
-  - template: ./templates/run-performance.yml
+      localTls: schannel
+      remoteTls: schannel
+      extraArgs: '-Publish -Record'
```

Comment:
By default, we probably shouldn't use the `-Record` option.

Reply:
Done